.github/workflows/ci.yml

```diff
@@ -13,7 +13,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.11", "3.12"]  # no particular need for 3.9 or 3.10
+        # Python >= 3.13 will fail with `ModuleNotFoundError: No module named 'lib2to3'`
+        # See: google/yapf#1258
+        python-version: ["3.8", "3.11", "3.12", "3.13"]  # no particular need for 3.9 or 3.10
```

Comment:
It still might be useful to add Py313 to the testing.

Reply:
Can probably drop 3.11 and 3.12 from the list. There's some special cases with 3.8 left in implementation code but 3.13 functionally tests everything >3.10 or greater.

There's also no special 3.9 code.